dev_tools/scripts/create_new_algorithm.sh

```diff
@@ -97,11 +97,13 @@ TEMPLATE_FILES=(
     "${name}_kernels.hpp"
     "${name}_kernels.cpp"
     "${name}_kernels.cpp"
-    "${name}_kernels.c*"
+    "${name}_*.[ch]*"
```

Comment:
Do we have a common naming scheme for CUDA header files? I see some `.cuh` and some `.hpp` files, so we might not catch all files with this pattern.

Reply:
I don't think we really have. I will have to take care of `cuh` down there in the code when I look for `hpp` I guess.

Reply:
Ah, I was thinking of regular expressions, not globs. So this should be fine